support-frontend/assets/helpers/tracking/quantumMetric.ts

```diff
@@ -128,6 +138,78 @@ function sendEventWhenReadyTrigger(sendEventWhenReady: () => void): void {
 	}
 }
 
+function sendEventAcquisitionDataFromQueryParamEvent(
+	acquisitionData: ReferrerAcquisitionData,
+): void {
+	void ifQmPermitted(() => {
+		const sendEventWhenReady = () => {
+			const source = acquisitionData.source;
+			const componentId = acquisitionData.componentId;
+			const componentType = acquisitionData.componentType;
+			const campaignCode = acquisitionData.campaignCode;
+			const referrerUrl = acquisitionData.referrerUrl;
+			const isRemote = acquisitionData.isRemote;
+
+			if (!source) {
+				return;
+			} else if (window.QuantumMetricAPI?.isOn()) {
+				sendEvent(
+					SendEventAcquisitionDataFromQueryParam.Source,
+					false,
+					source.toString(),
+				);
+			}
+			if (!componentId) {
```

Comment:
Perhaps these string not-truthy=return checks are not needed? Is something shorter like `if (componentId) {sendEvent...}` ok, you cant get zero through on a string/boolean so truthy should work I beleive?